src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/WT_FMC_Renderer.js

```diff
@@ -25,10 +25,17 @@ class WT_FMC_Renderer {
         this._fmc.parseContent = this.parseContent.bind(fmc);
 
         // init layout
+<<<<<<< HEAD
```

Comment:
It looks like all the merge conflict markers got committed as text, quite a bunch of them in the PR that I can see.

Reply:
Yeah, I think I forgot to `git add` my deconflicted files.   I am kinda surprised that git allowed me to commit conflict markers, I thought it was smarter than that.

The newer PR is better.